### ISSUE TITLE
Deploy Grafana from template

### ISSUE THREAD

**Issue by rayzhou2017** [kubesphere/kubesphere#1900]
Follow the guide https://kubesphere.com.cn/docs/v2.1/en/quick-start/one-click-deploy/, encounter the problems as below,

* [x] the default version of Grafana is the oldest one. Should be the latest one.
![WechatIMG36858](https://user-images.githubusercontent.com/28859385/74909938-4dc36c80-53f4-11ea-8986-00c4f9f0fd9b.jpeg)

* [ ] when deploy the oldest one, it won't work and no msg showing. It should throw error msg

**Comment by zheng1** [comment]
```bash
[root@master ~]# helm install grafana-0.2.0.tgz
Error: validation failed: unable to recognize "": no matches for kind "Deployment" in version "extensions/v1beta1"
```
/assign @pengcong06

**Comment by zheng1** [comment]
- [x] "the default version of Grafana is the oldest one. Should be the latest one." which is fixed in 2.1.1
- [ ] "when deploy the oldest one, it won't work and no msg showing. It should throw error msg", will be fixed in 3.0.0

**Comment by benjaminhuo** [comment]
Move this to 3.1 if it's not fixed in 3.0
